level2/level2_Dohyung/src/Application.java

```diff
@@ -0,0 +1,61 @@
+
+import java.util.Scanner;
+
+public class Application {
+    public static void askQuestion(){
+        Scanner sc = new Scanner(System.in);
+
+
+        while(true){
+            try{
+                System.out.println("덧셈할 문자를 입력해주세요");
+                String inputString = sc.nextLine();
+                if((!inputString.contains(","))&&(!inputString.contains(":"))){ // 구분자로 ,나 :가 없을 시 커스텀구분자 메소드실행
+                    customCalculate(inputString);
+                }
+                calculate(inputString);
+            }catch(IllegalArgumentException e){
+                System.out.println("잘못된 값을 입력하셨습니다.");
+                break;
```

Comment:
break;를 통해 시스템을 종료하고 계신데,
예외를 발생시켜 종료해주셔야 요구사항을 올바르게 지켰다고 볼 수 있습니다.
```suggestion
                throw new IllegalArgumentException("잘못된 값을 입력하셨습니다.");
```